ros-ocp/templates/secret-storage-credentials.yaml

```diff
@@ -1,9 +1,8 @@
-{{- $isOpenShift := eq (include "ros-ocp.isOpenShift" .) "true" }}
-{{- if not $isOpenShift }}
+{{- if eq (include "ros-ocp.isOpenShift" .) "false" }}
```

Comment:
I'd say that instead of copying the admin credentials, the user must define the location of the secret in the namespace it can use instead. This means the namespace must exist first, but then we now make sure that we are not responsible for the credentials anymore (using the admin might not be safe and also the user might not have access to the namespace). It is probable that the secret is created via gitops with Vault/SecretManagement before the chart is created. WDYT?